.markdownlinkcheck.json

```diff
@@ -3,6 +3,12 @@
     {
       "pattern": "^https://github.com/\\S+/\\S+/(issues|pull)/[0-9]+"
     },
+    {
+      "pattern": "^https://github.com/\\S+/\\S+/actions"
```

Comment:
Why don't similar links fail in other repos?

For example, each repo has GHA links like this in their README badges:

https://github.com/submariner-io/submariner/blob/devel/README.md

Reply:
I ask because the link linting has been helpful to catch actual failures with these GHA links, because they can change as workflow names change (and old workflows are deleted).